### ISSUE TITLE
Search results don't open in Safari

### ISSUE THREAD

**Issue by pbouffard** [JuliaComputing/MultiDocumenter.jl#55]
Chrome and Firefox do work. This is on Safari Version 16.5 (18615.2.9.11.4) on Ventura 13.4.

**Comment by mortenpi** [comment]
Is that all on Apple devices? It seems to work fine on Android with Chrome `113.0.5672.162`.

**Comment by pbouffard** [comment]
> Is that all on Apple devices? It seems to work fine on Android with Chrome `113.0.5672.162`.

Yes, that's all on an iPhone. Apple mandates that all browsers on iOS use WebKit. At least [for now.](https://9to5mac.com/2023/02/07/new-iphone-browsers/)

**Comment by pfitzseb** [comment]
Someone reported this before, but I can't repro this in Epiphany (which uses WebKitGTK 2.40.1):
![image](https://github.com/JuliaComputing/MultiDocumenter.jl/assets/6735977/5ff21fb6-893f-4d18-bc98-f2e559c85518)

Can't really do something about this without access to a recent Safari version.

**Comment by adrhill** [comment]
I can confirm that search results don't open on Safari Version 17.2.1 (19617.1.17.11.12).
